docs/source/llms/llama-index/notebooks/llama_index_workflow_tutorial.ipynb

```diff
@@ -0,0 +1,1081 @@
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "93515bbd-f648-4096-8030-1239a9180594",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from llama_index.core import Settings\n",
+    "from llama_index.llms.databricks import Databricks\n",
+    "\n",
+    "llm = Databricks(model=\"databricks-meta-llama-3-1-70b-instruct\")\n",
+    "Settings.llm = llm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "0534d0cb-c15e-4048-b32a-ef89aaa798d0",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "#### Option 3: Local LLM\n",
+    "\n",
+    "LlamaIndex also support locally hosted LLMs. Please refer to the [Starter Tutorial (Local Models)](https://docs.llamaindex.ai/en/stable/getting_started/starter_example_local/) for how to set them up."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "1b9bc91b-d9e2-42f2-92be-8bbbd73c2062",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Create an MLflow Experiemnt\n",
+    "\n",
+    "*Skip this step if you are running this tutorial on Databricks Notebook. An MLflow experiment is automatically set up when you created the notebook.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "387ef167-c2f4-496f-9197-3d62cb20f4eb",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import mlflow\n",
+    "\n",
+    "mlflow.set_experiment(\"MLflow LlamaIndex Workflow Tutorial\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "c2506f73-6975-48ae-9f5e-0789362dd48b",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Define tools\n",
+    "\n",
+    "The agents access with various functions and resources via `tool` objects. In this example, we define the simplest possible math tools `add` and `multiply` based on Python functions. For the real application, you can create arbitral tools such as vector search retrieval, web search, or even calling another agent as a tool. Please refer to the [Tools documentation](https://docs.llamaindex.ai/en/stable/module_guides/deploying/agents/tools/) for more details.\n",
+    "\n",
+    "\n",
+    "Please ignore the `### [USE IN MODEL]` comment at the beginning of some cells like below. This will be used in the later step in this tutorial!"
```

Comment:
```suggestion
    "Please ignore the `### [USE IN MODEL]` comment at the beginning of some cells like below. This will be used in later steps in this tutorial!"
```